aws/lambda/pytorch-auto-revert/pytorch_auto_revert/__main__.py

```diff
@@ -113,6 +113,15 @@ def get_opts() -> argparse.Namespace:
             "Revert mode: skip, log (no side effects), run-notify (side effect), or run-revert (side effect)."
         ),
     )
+    workflow_parser.add_argument(
```

Comment:
this functionality should not be a flag on revert mode, instead a separate entry point that reads from the database and creates this html.

this is because I don't want to couple the autorevert logic to the html generation logic. 

This is not safe for a service, it makes more sense for those to be separated entities/actions.

Reply:
wdym?

there **IS** a separate endpoint: `hud` subcommand that reads and prints from the database

the flag to the autorevert just reuses the functionality to optionally dump the results of the run as html, it's a convenience